examples/dem/3d-plate-discharge/data/output_asc-lb/dummy-file

```diff
@@ -0,0 +1,2 @@
+Dummy file to ensure folder exists
```

Comment:
I think these dummy files are not required anymore

Reply:
It is best to keep them for portability accross mac OS and other OSes